src/components/ListWraper/ListItem/ListItem.js

```diff
@@ -3,30 +3,30 @@ import React, { Component } from 'react';
 
 class ListItem extends Component {
 
-  findindexKey = ( array, id ) => array.findIndex( object => object.id === id );
-  
+  findindexKey = (array, id) => array.findIndex(object => object.id === id);
+
   removeRow = () => {
     const {
       id,
       state
     } = this.props;
 
-    delete state.users[ this.findindexKey(state.users, id) ];
+    delete state.users[this.findindexKey(state.users, id)];
```

Comment:
ta zmienna state, to jest słowko zarezerowane dla wewnetrznej implementacji Reacta, nie powinnienes tak nazywac zmiennej. -> nazwij ją data jesli to trzyma dane czy coś

destukturyzacja users z data
```
   const {
      id,	      
      data: {users}
    } = this.props;	
```

znając index elementu od usunięcie elementu uzyj splice
`https://developer.mozilla.org/pl/docs/Web/JavaScript/Referencje/Obiekty/Array/splice`

index tego elementu przpypisz do stałej
`
const rowToRemoveIndex = this.findIndex();
`